doc/whatsnew/2/2.15/index.rst

```diff
@@ -61,3 +61,9 @@ Internal changes
 * ``pylint.testutils.primer`` is now a private API.
 
   Refs #6905
+
+* Fixed an issue where it was impossible to update functional tests output when the existing
+  output was impossible to parse. Instead of raising an error we raise a warning message and
+  let the functional test fail with a default value.
+
+  Closes #6891
```

Comment:
```suggestion
```

Reply:
I think if there is no issue then we need to put the PR or there's no way to easily see the discussion around the PR otherwise. (You would have to git blame the changelog text...)

Reply:
Let's use `ref` then. Closes feels weird.

Reply:
```suggestion
  Refs #6891
```